src/compiler/msvc.rs

```diff
@@ -391,10 +391,19 @@ msvc_args!(static ARGS: [ArgInfo<ArgData>; _] = [
     msvc_take_arg!("doc", PathBuf, Concatenated, TooHardPath), // Creates an .xdc file.
     msvc_take_arg!("errorReport:", OsString, Concatenated, PassThroughWithSuffix), // Deprecated.
     msvc_take_arg!("execution-charset:", OsString, Concatenated, PassThroughWithSuffix),
+    msvc_flag!("experimental:external", PassThrough),
     msvc_flag!("experimental:module", TooHardFlag),
     msvc_flag!("experimental:module-", PassThrough), // Explicitly disabled modules.
     msvc_take_arg!("experimental:preprocessor", OsString, Concatenated, PassThroughWithSuffix),
     msvc_take_arg!("external:I", PathBuf, CanBeSeparated, ExternalIncludePath),
+    msvc_flag!("external:W0", PassThrough),
```

Comment:
Could you please add comments explaining why we are adding these flags?
thanks

Reply:
@sylvestre Adding those flags are required because otherwise sccache treats them wrongly, resulting in false-positive cache misses.

Reply:
Can you elaborate on what it means for "sccache treats them wrongly"?
Is this an issue with an equivalent of `-Werror` where different warning levels may change whether or not a compile passes or succeeds?

If you can provide this information in the commit message, that'd be ideal :)